llvm-generator/lib/Minicute/Transpilers/Generator.hs

```diff
@@ -33,6 +33,59 @@ generateMachineCodeSc (Identifier binder, _, expr)
 generateMachineCodeE :: GMachineExpression -> IRBuilderT ModuleBuilder ()
 generateMachineCodeE = go []
   where
+    go vStack (IMakeInteger v : insts@(_ : _)) = do
+      sName <- load (AST.ConstantOperand constantAddrStackPointer) 0
+      sName' <- gep sName [operandInt 32 1]
+      nName <- call (AST.ConstantOperand constantCreateNodeNInteger) [(operandInt 32 (toInteger v), [])]
+      store nName 0 sName'
+      store sName' 0 (AST.ConstantOperand constantAddrStackPointer)
+
+      go vStack insts
+    go vStack (IMakeApplication : insts@(_ : _)) = do
+      sName <- load (AST.ConstantOperand constantAddrStackPointer) 0
+      sName' <- gep sName [operandInt 32 (negate 1)]
```

Comment:
Use `(-1)` instead.

Reply:
Is there any special reason?
As I know, Many Haskell articles suggest that I use `negate` instead of prefix `-` operator.

Reply:
@ENvironmentSet Could you give me some examples? I haven't seen one. If that's true, I need to add a hlint rule for that, I guess.

Reply:
@ENvironmentSet I added the rule. Now I'll resolve all the reviews I made.